config/jobs/kubernetes-sigs/kueue/kueue-release-blocking.yaml

```diff
@@ -0,0 +1,187 @@
+periodics:
+  - interval: 24h
+    name: periodic-kueue-test-unit-main
+    annotations:
+      testgrid-dashboards: sig-scheduling
+      testgrid-tab-name: periodic-kueue-test-unit-main
+      description: "Run periodic kueue unit tests"
+      testgrid-num-columns-recent: '6'
+      testgrid-num-failures-to-alert: "1"
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    extra_refs:
+      - org: kubernetes-sigs
+        repo: kueue
+        base_ref: master
+        path_alias: kubernetes-sigs/kueue
+    decorate: true
+    decoration_config:
+      timeout: 1h
+    spec:
+      containers:
+        - image: golang:1.19
+          env:
+            - name: GO_TEST_FLAGS
+              value: "-race -count 3"
+          command:
+            - make
+          args:
+            - test
+  - interval: 24h
+    name: periodic-kueue-test-integration-main
+    annotations:
+      testgrid-dashboards: sig-scheduling
+      testgrid-tab-name: periodic-kueue-test-integration-main
+      description: "Run periodic kueue test-integration"
+      testgrid-num-columns-recent: '6'
+      testgrid-num-failures-to-alert: "1"
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    extra_refs:
+      - org: kubernetes-sigs
+        repo: kueue
+        base_ref: master
+        path_alias: kubernetes-sigs/kueue
+    decorate: true
+    decoration_config:
+      timeout: 1h
+    spec:
+      containers:
+        - image: golang:1.19
+          command:
+            - make
+          args:
+            - test-integration
+  - interval: 24h
+    name: periodic-kueue-test-e2e-main-1-23
```

Comment:
Remove this one too. 1.23 is out of support now.
We can add 1.26 in a follow up.